### PR TITLE
PCL Refactoring

### PR DESCRIPTION
**Pull request by kroniak**
- delete PCL refs
- add support netstandard 1.0 for Flurl
- add support netstandard 1.1 for Flurl.Http with API non changeable.
- bump Nunit test runner version

### CODE REVIEW

**Comment by tmenier** [comment]
@kroniak Here are my thoughts, but I consider you an expert on platform targeting and I value your opinions:

1. I was hoping we could factor out PCLStorage as part of this. It hasn't been updated in 2.5 years, and [System.IO.FileSystem](https://www.nuget.org/packages/System.IO.FileSystem/) has since been released out of band, targeting `netstandard1.3` and several older Xamarin platforms. Do we need to keep PCLStorage just for `netstandard1.1` support? What are our options?

2. Should we include a `net46` target? The reason would be that .NET 4.6.x projects will get the `net45` version, even though `netstandard1.3` contains newer implementations. (I recently confirmed this [here](https://github.com/NuGet/Home/issues/358#issuecomment-315881625).)

3. I think there should be one package tester per target platform, i.e. one that should get the `netsandard1.3` version, one that should get the `net45` version, etc. Do you agree?

File: src/Flurl.Http/NoOpTask.cs [hunk 1]
```diff
@@ -4,10 +4,6 @@ namespace Flurl.Http
 {
```
Comment (tmenier) [inline src/Flurl.Http/NoOpTask.cs]: Let's drop this class. It only existed to abstract the platform-switching noise.

Comment (kroniak) [inline src/Flurl.Http/NoOpTask.cs]: As you wish

**Comment by kroniak** [comment]
@tmenier 
1. Yes, PCLStorage needs only for download api for old platform (net std 1.1). We can drop it and remove all download API from Flurl.Http for net std 1.1 as https://github.com/tmenier/Flurl/pull/200/commits/f98de654810da4fd2f691b276a3fc43528576952

2. No, I dont think that dll in netstd is faster or better than dll in net45.
3. Yes. One for net45, one for netcore.

**Comment by tmenier** [comment]
So if we drop PCLStorage and netstd1.1 we would [only lose win8 and wp8.1](https://docs.microsoft.com/en-us/dotnet/standard/net-standard). Not sure how many that would affect, but I guess there's no harm in keeping it except for a very little bit of code bloat, and it is well separated. It sounds like you are in favor of keeping it? I'll accept your PR, and we can always change our minds later.

**Comment by kroniak** [comment]
@tmenier  Yes, I want to keep it yet.

**Pull request merged**